### PR TITLE
Fix basic game loading

### PR DESCRIPTION
**Pull request by ishermandom**
I think we had errors in the `states.inc.php` file – single vs. double quotes – that were breaking basic parsing. The game seems playable with these changes landed.

Not all of the changes are critical, a bunch are just various cleanups while I was trying to figure out what was going wrong.

### CODE REVIEW

**Review by jasonptm — COMMENTED** [review]
Re: tiebreaker, I think this deserves a small amount more discussion in person (e.g. do we care if win-by-two has a theoretically unbounded play time?), but we can just go with the "do something reasonable" first approximation and update later.

**Review by jasonptm — APPROVED** [review]

File: version.php [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<?php 
+<?php
 // Please do not modify this file.
```
Comment (sdspikes) [inline version.php:2]: You modified the do not modify file! :P

Comment (ishermandom) [reply version.php:2]: O=)

I still don't actually know whether any of the names matter! But I created a new new project to see what all the defaults were, and then tried to backfill for this project... seems like it worked :)

Comment (jasonptm) [reply version.php:2]: I think it's probably fine in this case, but in my experience "seems like it worked" is a real "hold my beer" of a phrase: it presages disaster.

**Pull request merged**